java/cuvs-java/src/main/java/com/nvidia/cuvs/GPUInfo.java

```diff
@@ -18,11 +18,10 @@
 /**
  * Contains GPU information
  *
- * @param gpuId             id of the GPU starting from 0
- * @param name              ASCII string identifying device
- * @param freeMemory        returned free memory in bytes
- * @param totalMemory       returned total memory in bytes
- * @param computeCapability the compute capability of the device
+ * @param gpuId                    id of the GPU starting from 0
+ * @param name                     ASCII string identifying device
+ * @param totalDeviceMemoryInBytes total device memory in bytes
+ * @param computeCapability        the compute capability of the device
  */
 public record GPUInfo(
-    int gpuId, String name, long freeMemory, long totalMemory, float computeCapability) {}
+    int gpuId, String name, long totalDeviceMemoryInBytes, float computeCapability) {}
```

Comment:
Good change. I prefer the new name: `totalDeviceMemoryInBytes`.

Question: Do we no longer require the free memory reading?  At some point in the future, one wonders if that's something that might be useful in scheduling decisions.

Not a problem, of course.  That's easy to add later on.

Reply:
Oh, wait, I see.  Please ignore.  This info has been moved to `CuVSResourcesInfo`.

Reply:
Yes, I moved transient information/information you want to poll at a given time (like the amount of free memory) to a separate record + function to get them